config/environments/development.rb

```diff
@@ -38,4 +38,13 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.paperclip_defaults = {
+      storage: :s3,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.